custom-vu/src/main/kotlin/jces1209/vu/page/customizecolumns/CustomizeColumns.kt

```diff
@@ -0,0 +1,9 @@
+package jces1209.vu.page.customizecolumns
+
+interface CustomizeColumns {
```

Comment:
What about to specify different names for action and page component?
e.g. CustomizeColumns  - Action, ColumnsEditor - Page

Reply:
renamed